src/main/scala/ai/privado/exporter/CollectionExporter.scala

```diff
@@ -167,6 +198,23 @@ class CollectionExporter(cpg: Cpg) {
     )
   }
 
+  def processByLiteralId(
```

Comment:
@hiteshbedre Also check if possible to have a single function processById, instead of `processByParameterId`, `processByLocalVariableId` and `processByLiteralId`

Reply:
Need a parent class which has properties of AstNode also supports _.method_ method over it, not found as of now. Will spend some extra time on this soon.